config/tracing.go

```diff
@@ -0,0 +1,40 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"slices"
+)
+
+type TracingConfig struct {
+	Enabled                 bool                 `yaml:"enabled" default:"false"`
+	Attributes              map[string]string    `yaml:"attributes"`
+	Opentelemetry           *OpenTelemetryConfig `yaml:"opentelemetry"`
+	ServiceName             string               `yaml:"service_name" default:"WebhookX"`
+	CapturedRequestHeaders  []string             `yaml:"captured_request_headers"`
+	CapturedResponseHeaders []string             `yaml:"captured_response_headers"`
+	SafeQueryParams         []string             `yaml:"safe_query_params"`
+	SamplingRate            float64              `yaml:"sampling_rate" default:"1.0"`
+}
+
+type OpenTelemetryConfig struct {
+	Protocol OtlpProtocol      `yaml:"protocol" envconfig:"PROTOCOL" default:"http/protobuf"`
+	Endpoint string            `yaml:"endpoint" envconfig:"ENDPOINT" default:"http://localhost:4318/v1/metrics"`
+	Headers  map[string]string `yaml:"headers,omitempty"`
```

Comment:
Can we remove it as it can be configured via Otel Envs? Or do you think this is required?

Reply:
it's necessary, webhookx should start up with a simple default config rather than letting user gain a deeper understanding about OTEL Envs.
It can be override by OTEL Envs if users have more requirements of tracing